internal/controller/verticalpodautoscaler/admission_plugin.go

```diff
@@ -13,12 +15,20 @@ type AdmissionPluginArg string
 // These constants represent the vertical-pod-autoscaler arguments used by the
 // operator when processing VerticalPodAutoscalerController resources.
 const (
+	KubeAPIQPSArg    AdmissionPluginArg = "--kube-api-qps"
+	KubeAPIBurstArg  AdmissionPluginArg = "--kube-api-burst"
 	TLSCertFileArg   AdmissionPluginArg = "--tls-cert-file"
 	TLSKeyFileArg    AdmissionPluginArg = "--tls-private-key"
 	TLSCACertFileArg AdmissionPluginArg = "--client-ca-file"
 	WebhookTimeout   AdmissionPluginArg = "--webhook-timeout-seconds"
 )
 
+func CheckArgExists(args []string, arg AdmissionPluginArg) bool {
```

Comment:
How do you feel about `ArgExists` instead of `CheckArgExists`?  And you could also probably stuff it in util, but I don't know that it's hurting anything living where it is.

Reply:
Sure sounds good, always open to better variable/function names :D